src/pyhf/constraints.py

```diff
@@ -1,25 +1,41 @@
 from . import get_backend, default_backend
 from . import events
+from .paramview import ParamViewer
 
 
 class gaussian_constraint_combined(object):
-    def __init__(self, pdfconfig):
+    def __init__(self, pdfconfig, batch_size=None):
```

Comment:
set default batch size to 1

Reply:
we will need a check for batch size = 0 (or negative) perhaps.

Reply:
The idea is that as a pdf, there are 2 possibilities

* logpdf() returns a scalar ( shape `()`)  (batch_size = None)
* logpdf() returns a tensor ( shape `(batch_size,)` )

this is in preparation for moving to e.g. tensorflow_probability and pytorch.distributions. So this

Reply:
ok